### PR TITLE
Fix set_changelog step for tvOS releases

### PR DESCRIPTION
**Pull request by dcordero**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md) and run `bundle exec fastlane lint` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

As @carolanitz detected, the step `update_changelog` does not work as expected for tvOS releases, and it actually tries to set the changelog of the iOS application instead.

Since fastlane 2.90.0 🚀, there is new parameter `platform` available in the set_changelog action which allows fixing this issue.

This PR set this new parameter and, in addition to that, it converts the changelog file to a more standard and easier to parse format.

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -95,12 +95,20 @@ end
...
   platform = options[:platform]
-  # Splits the News by -------- get out the top notes
-  changelog = File.read('../Docs/NEWS').split('-----------')[1].split('-----------').first
-  temp_changelog = changelog.split(platform)
-  temp_changelog = changelog.split('tvOS') if temp_changelog.count <= 1
-  changelog = temp_changelog[0..-2].join.strip
-  set_changelog(app_identifier: 'org.videolan.vlc-ios', changelog: changelog, username: '*', team_name: 'VideoLAN')
+  version = options[:version]
+
+  regex = /## #{platform} \[#{version}\](.*?)^##/m
```
Comment (carolanitz) [inline fastlane/Fastfile]: that is sooo much better than what I had!!

**Comment by carolanitz** [comment]
This looks great and I like the News format better than the old one

File: fastlane/Fastfile (hunk 1 above)
Comment (dcordero) [reply fastlane/Fastfile]: I cheated changing the format of the changelog, the previous format was much harder the parse 😉

**Comment by dcordero** [comment]
@fkuehne done ✅. 

I changed them by - to make the document markdown compatible, but there is actually no need at all to make it a md file. I do agree that listing with with · looks much better than using minuses symbols.

File: Docs/NEWS [hunk 2]
```diff
@@ -1,114 +1,124 @@
...
------------
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## tvOS [Unreleased]
+
+## iOS [Unreleased]
```
Comment (dcordero) [inline Docs/NEWS]: What do you think about these `unreleased` sections? 

The purpose of them is adding entries to the changelog progressively as soon as feature, fixes, etc... are implemented, getting at the end everything listed at the moment of releasing.

Does this workflow makes sense for vlc-ios?

Comment (carolanitz) [reply Docs/NEWS]: I think they will probably disappear soon since I tend to just copy paste from the line below :D

**Comment by carolanitz** [comment]
merged with 84b47471dd1c446071cc2d5c0785f16357735b85

**Pull request closed without merging**